src/main/java/rip/bolt/ingame/ranked/RankedManager.java

```diff
@@ -109,17 +112,43 @@ private void updateMatch(BoltMatch match) {
       return;
     }
 
-    List<MatchPlayer> players =
+    MatchManager mm = PGM.get().getMatchManager();
+
+    List<UpdateRecord> updates =
         match.getTeams().stream()
             .map(Team::getParticipations)
             .flatMap(Collection::stream)
             .map(Participation::getUser)
-            .map(rankManager::notifyUpdates)
-            .filter(Objects::nonNull)
+            .map(
+                u ->
```

Comment:
same here no real reason to use `u`, replacing it with `user` and formatting puts it on multiple lines which makes it look a lot neater